the-blue-alliance-ios/View Controllers/Events/EventViewController.swift

```diff
@@ -58,11 +61,12 @@ class EventViewController: ContainerViewController {
             matchesViewController.matchSelected = { match in
                 self.performSegue(withIdentifier: "MatchSegue", sender: match)
             }
-        } else if segue.identifier == "EventMatchesEmbed" {
```

Comment:
Let's not remove the Matches embed segue... we need to make sure we still setup the matches VC